modules/ask_va_api/app/lib/ask_va_api/inquiries/payload_builder/veteran_profile.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+module AskVAApi
+  module Inquiries
+    module PayloadBuilder
+      class VeteranProfile
+        include SharedHelpers
+
+        class VeteranProfileError < StandardError; end
+
+        attr_reader :inquiry_params, :inquiry_details
+
+        def initialize(inquiry_params:, inquiry_details:)
+          @inquiry_params = inquiry_params
+          @inquiry_details = inquiry_details
+          @translator = Translator.new
+        end
+
+        def call
+          base_profile
+            .merge(service_info)
+        end
+
+        private
+
+        def base_profile
+          {
+            FirstName: inquiry_params.dig(:about_the_veteran, :first),
+            MiddleName: inquiry_params.dig(:about_the_veteran, :middle),
+            LastName: inquiry_params.dig(:about_the_veteran, :last),
+            PreferredName: inquiry_params.dig(:about_the_veteran, :preferred_name),
+            Suffix: @translator.call(inquiry_params.dig(:about_the_veteran, :suffix)),
+            Country: nil,
+            Street: inquiry_params.dig(:about_the_veteran, :street),
+            City: inquiry_params.dig(:about_the_veteran, :city),
+            State: state_data,
+            ZipCode: inquiry_params[:veteran_postal_code],
+            DateOfBirth: inquiry_params.dig(:about_the_veteran, :date_of_birth)
+          }
+        end
```

Comment:
for increased readability consider `extracting inquiry_params.dig(:about_the_veteran)` as a variable

```ruby
def base_profile
  veteran = inquiry_params.dig(:about_the_veteran)
    {
       FirstName: veteran[:first],
       MiddleName: veteran[:middle]